Author: Prateek Ganguli <prateek.ganguli@gmail.com>
Last-Update: 2021-04-16
Description: Fix src paths in the build script
 This patch modifies build.gradle so that it can find the source
 files in external/jsilver, rather than the default of repo root.
--- a/external/jsilver/build.gradle
+++ b/external/jsilver/build.gradle
@@ -9,11 +9,11 @@
 sourceSets {
     main {
         java {
-            srcDirs = ['src']
+            srcDirs = ['external/jsilver']
             exclude 'com/google/clearsilver/jsilver/benchmark/*.java'
         }
         resources {
-            srcDirs = ['src']
+            srcDirs = ['external/jsilver']
             include '**/*.dat'
         }
     }
@@ -23,4 +23,4 @@
     compile 'com.google.guava:guava:debian'
 }
 
-apply from: 'debian/generatePom.gradle'
\ No newline at end of file
+apply from: '../../debian/external/jsilver/generatePom.gradle'
